base/serialize.jl

```diff
@@ -39,7 +39,7 @@ const TAGS = Any[
     :a, :b, :c, :d, :e, :f, :g, :h, :i, :j, :k, :l, :m, :n, :o,
     :p, :q, :r, :s, :t, :u, :v, :w, :x, :y, :z,
     :add_int, :sub_int, :mul_int, :add_float, :sub_float,
-    :mul_float, :unbox, :box,
+    :mul_float, :bitcast, :box,
```

Comment:
I'm assuming someone will change it to `reservedN` at some point or reuse it, but that keeping it in place keeps the indexing the same for `ser_version`.